docs/dev/README.md

```diff
@@ -22,6 +22,7 @@ If this is your first time using zkSync, we recommend that you kick off at the b
 - [Account abstraction](./developer-guides/aa.md) - Learn about account abstraction.
 - [Fee mechanism](./developer-guides/transactions/fee-model.md) - A quick overview of the zkSync fee structure.
 - [Bridging of funds](./developer-guides/bridging/bridging-asset.md) - A brief intro on token bridging.
+    - [Add tokens to bridge](./developer-guides/bridging/bridging-asset.md#add-tokens-to-bridge) - Add your token to the bridge
```

Comment:
```suggestion
    - [Add tokens to bridge](./developer-guides/bridging/bridging-asset.md#add-tokens-to-bridge) - Add your token to the bridge.
```

Reply:
Did you see Ramon's comment? https://most.matter-labs.io/matterlabs/pl/4fr3n3eieffdped5tjbphgpnwh

Reply:
Hi, I made another change. I don't like the word Default either. It's confusing. What is a Default bridge? Is it the same as the standard bridge, I suspect so. If it is, I would change the title **Default bridge** to **Bridge addresses** which is clearer. I'll add that too now.

Reply:
I've updated it @kailash-manasarovar